src/executables/sysrepo-plugind.c

```diff
@@ -515,6 +519,9 @@ main(int argc, char **argv)
 
             rc = EXIT_SUCCESS;
             goto cleanup;
+        case 'p':
+            pidfile = strdup(optarg);
```

Comment:
Totally redundant and leaking `strdup()`.

Reply:
Is it really redundant? `optarg` is a global variable defined by `getopt`, and I don't want to just copy the pointer as it will most likely change over time. I did miss the `free(pidfile)` call, but i'ts been added now.

Reply:
The pointer does change but the string itself does not, the parsed parameters are kept for the whole lifetime of the process. It is safe to use it directly.

Reply:
Oh, ok, I did not know that!